### PR TITLE
Add clarity to README

### PR DESCRIPTION
**Pull request by ellsclytn**
- Use the correct build badge
- Explain how Boiler Room Runner is used in BRB
- Formatting, semantics (lame)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Boiler Room Builder
 
-[![Build status](https://badge.buildkite.com/35edf858022bf6c8ec20dc8a3433348f4a268d772991e2c913.svg)](https://buildkite.com/everyday-hero/boiler-room-runner-tests)
+[![Build status](https://badge.buildkite.com/3b53417ee3fb24b145098c129e6b31cc2ddc42d39d98ab0b52.svg?style=flat-square)](https://buildkite.com/everyday-hero/boiler-room-builder-tests)
 
-A tool-chain for universal react apps
+Create React apps without writing your own build configuration. With little effort you will be able to `build`, `test`, `lint` and `serve` like a pro.
```
Comment (aabrook) [inline README.md]: You write the react app, boiler-room-builder creates the build configuration

Comment (alexander-bobin) [inline README.md]: "You write the React app, Boiler Room Builder (BRB) handles the build configuration."

File: README.md [hunk 2]
```diff
@@ -29,15 +28,16 @@ $ npm i boiler-room-builder --save-dev
...
 ### Create your `source/client.js` and `source/server.js` files
 
-Refer to the [basic example](https://github.com/everydayhero/boiler-room-builder/tree/master/examples/basic), to see what a boiler-room-builder setup looks like without any external libs.
+Refer to the [basic example](https://github.com/everydayhero/boiler-room-builder/tree/master/examples/basic), to see what a boiler-room-builder setup looks like without any external libraries.
 
-**Server.js**
+#### Server.js
 
-If you use [Boiler room runner](https://github.com/everydayhero/boiler-room-runner#in-your-serverjs-file) to bootstrap your app, this will be simpler :).
+If you use [Boiler room runner](https://github.com/everydayhero/boiler-room-runner#in-your-serverjs-file) to bootstrap your app, this will be simpler :simple_smile:).
```
Comment (alexander-bobin) [inline README.md]: Would prefer to move mention of BRR down instead of starting with it.

File: README.md [hunk 3]
```diff
@@ -56,9 +56,17 @@ export default function ({ assets = [] }) {
 
 Why return a function like this? The exported function is passed into a Promise chain, which allows you to perform some async setup for your app before we try to call it to render any html.
 
-**Client.js**
+#### Client.js
+
+You can put pretty much anything in your client file, builder wont try to require and run it. It is however recommended that you use [Boiler Room Runner](https://github.com/everydayhero/boiler-room-runner#in-your-clientjs-file). It's got what you need.
```
Comment (alexander-bobin) [inline README.md]: "builder" to BRB.

Let's move the recommendation for BRR out to it's own area maybe.

File: README.md [hunk 4]
```diff
@@ -56,9 +56,17 @@ export default function ({ assets = [] }) {
...
+
+### Boiler Room Runner's Purpose in Builder
+
+Though BRR is not strictly necessary in any BRB project, BRR is included to provide a default document, in case you don’t provide your own exported `renderDocument` in `server.js` .
```
Comment (alexander-bobin) [inline README.md]: We are using BRR as an acronym before doing the intro. Should be "Boiler Room Runner (BRR)" somewhere

**Comment by bradparker** [comment]
LGTM @alexander-bobin

**Pull request merged**